src/lib.rs

```diff
@@ -1,14 +1,48 @@
 use std::fmt;
 
-/// TODO...
+/// A trait that can be used for creating and converting between valid [numerical](https://www.w3.org/TR/2018/REC-css-color-3-20180619/#numerical) color values.
```

Comment:
maybe "for converting between different color models and performing various transformations on them."?